e2e/packages/ads/src/index.tsx

```diff
@@ -1,5 +1,10 @@
+/* eslint-disable */
+/*
+ * TSDocs will be added in this branch:
+ * https://github.com/frontity/frontity/tree/package-name
```

Comment:
Is this correct? I see that the last commit on that branch was 8 days ago.

Even if yes, I would say that we shouldn't keep track of tasks this way because it's asking for trouble once it's merged and someone forgets 😅

Reply:
You're right, it's not a good approach. I should've copied the TSDocs from that branch 👍